rdkit/Chem/UnitTestRegistrationHash.py

```diff
@@ -276,13 +276,15 @@ def test_enhanced_stereo_canonicalizer(self):
             ),
         )
         for mols_smis in groups:
-            print()
             csmis = set()
             for smi in mols_smis:
                 mol = Chem.MolFromSmiles(smi)
-                csmi = Chem.CanonicalizeEnhancedStereo(mol)
+                Chem.CanonicalizeEnhancedStereo(mol)
+                csmi = Chem.MolToCXSmiles(mol)
                 csmis.add(csmi)
             self.assertEqual(len(csmis), 1)
+            self.assertNotEqual(list(csmis)[0], None)
```

Comment:
I would suggest replacing this test with something inside the loop (i.e. above line 284):
```
                self.assertIsNotNone(csmi)
```